config/accounts_test.go

```diff
@@ -771,20 +771,34 @@ func TestAlternateBidderCodes_IsValidBidderCode(t *testing.T) {
 			wantErr:     errors.New(`alternateBidderCodes not defined for adapter "pubmatic", rejecting bids for "groupm"`),
 		},
 		{
-			name: "account.alternatebiddercodes config enabled but adapter config has allowedBidderCodes list empty or not defined",
+			name: "account.alternatebiddercodes config enabled but adapter config is disabled",
 			args: args{
 				bidder:          "pubmatic",
 				alternateBidder: "groupm",
 			},
 			fields: fields{
 				Enabled: true,
 				Bidders: map[string]AdapterAlternateBidderCodes{
-					"pubmatic": {},
+					"pubmatic": {Enabled: false},
 				},
 			},
 			wantIsValid: false,
 			wantErr:     errors.New(`alternateBidderCodes disabled for "pubmatic", rejecting bids for "groupm"`),
 		},
+		{
+			name: "account.alternatebiddercodes and adapter config enabled but adapter config does not have allowedBidderCodes defined",
+			args: args{
+				bidder:          "pubmatic",
+				alternateBidder: "groupm",
+			},
+			fields: fields{
+				Enabled: true,
+				Bidders: map[string]AdapterAlternateBidderCodes{
+					"pubmatic": {Enabled: true},
```

Comment:
Nitpick: I recommend using the following declaration to make it clear to the reader this is the case for a nil `AllowedBidderCodes`:

```
"pubmatic": {
    Enabled:            true,
    AllowedBidderCodes: nil
},
```